### ISSUE TITLE
Empty "Overwrite in project theme" pop-up message when overwriting .phtml

### ISSUE THREAD

**Issue by zehsebasl** [magento/magento2-phpstorm-plugin#946]
**Describe the bug** (*)
When overwriting a phtml file, the theme selector is empty.  

**To Reproduce** (*) 
> Go to vendor
> Open Module Catalog
> View > frontend > templates >  product > list.phtml 
> right-click  > "Overwrite this template in a project theme" > *click

**Expected behavior** (*) 
I Expected to be able to select a theme. The theme resides within app/code/design/{vendor}/{theme-name}/theme.xml

**Screenshots**
![image](https://user-images.githubusercontent.com/45463850/152009427-d93baccb-ddbd-4880-9223-8c59a2d39aaa.png)

**Please complete the following information:** (*) 

 - OS: Mac Big Sur
 - PhpStorm/Intellij version: 2021.3.1
 - Plugin Version: 4.2.2

**Additional context**

I used Mark Susht Docker script for local development. 
https://github.com/markshust/docker-magento

**Comment by bohdan-harniuk** [comment]
Hello, @zehsebasl!

Thank you for your report!
We will try to fix this issue as soon as possible.

Will you be able to test in your local environment whether the fix works by installing alpha release?

Regards,

**Comment by zehsebasl** [comment]
> Hello, @zehsebasl!
> 
> Thank you for your report! We will try to fix this issue as soon as possible.
> 
> Will you be able to test in your local environment whether the fix works by installing alpha release?
> 
> Regards,

I will look into this and come back at you.

**Comment by bohdan-harniuk** [comment]
Thank you, @zehsebasl!

I will ping you just after we fix this and publish new alpha release!

Regards,

**Comment by Iamwade** [comment]
@magento I am working on this

**Comment by zehsebasl** [comment]
> Thank you, @zehsebasl!
> 
> I will ping you just after we fix this and publish new alpha release!
> 
> Regards,

Thank you for helping me @bohdan-harniuk. 
I'm sorry. I followed the steps to run 4.3.0-develop locally, but it seems I require Intellij Ultimate. I don't have that, just the regular Phpstorm. So therefor, I cannot test branch 4.3.0 at this moment.

**Comment by zehsebasl** [comment]
Thank you @Iamwade 

Our theme registration: 

<?php
\Magento\Framework\Component\ComponentRegistrar::register(
    \Magento\Framework\Component\ComponentRegistrar::THEME,
    'frontend/{vendor}/{theme-name1/3}_{theme-name2/3}_{theme-name3/3}',
    __DIR__
);

Writing this down, I can imagine that having "_" in the theme "componentName" is not wise. But it's technically allowed as the theme is currently active with this componentName. 

So I tested with the following registration.php as well:

<?php
\Magento\Framework\Component\ComponentRegistrar::register(
    \Magento\Framework\Component\ComponentRegistrar::THEME,
    'frontend/{vendor}/{theme-name}',
    __DIR__
);

But the results are the same.  The model dropdown for selecting a theme keeps being empty.

**Comment by Iamwade** [comment]
@zehsebasl 
Thanks for your screenshots.
![image](https://user-images.githubusercontent.com/33068778/152021274-f1b391a2-8e83-4641-b9e0-ff14790d40e3.png)

Is the first letter in the theme title with a capital letter?

**Comment by zehsebasl** [comment]
@Iamwade 

Yes, That is correct. Both in the registration and the folder name contain capitalization. 

I tried mixing the typography to see if this resolves the issue, removing capitalization in the registration.php and folder name did not resolve the issue unfortunately. 

I tried opening other projects we have and in one particular project, the dropdown worked. 
So i'm going to compare the theme files/structure to see why one is working and the other is not. 

This proves to me that there is no "issue" with the plugin but that the problem resides within our own configuration of that project. 
![image](https://user-images.githubusercontent.com/45463850/152119607-307a0685-f218-41a9-8776-c399414383c3.png)

I will close this issue; and report the difference when I have more time so that other people having this issue can see what the difference was as well.
